app/src/test/java/org/simple/clinic/sync/DataSyncTest.kt

```diff
@@ -6,11 +6,16 @@ import io.reactivex.Completable
 import io.reactivex.Single
```

Comment:
Please check the commit message description.
"This test was ~~not~~ failing earlier..."

Reply:
Oh wait, the commit message is correct. We use `RxErrorsRule` because there are cases when an Rx chain can fail but the test continues to pass. This was one of those cases where the test was broken because a stub was missed, but it kept passing because we had missed adding the rule.

Reply:
Ah, okay.